lib/timescaledb/rails/models/continuous_aggregate.rb

```diff
@@ -13,6 +13,18 @@ class ContinuousAggregate < ::ActiveRecord::Base
 
       has_many :jobs, foreign_key: 'hypertable_name', class_name: 'Timescaledb::Rails::Job'
 
+      # Manually refresh a continuous aggregate.
+      #
+      # @param [String] view
+      # @param [DateTime] start_time
+      # @param [DateTime] end_time
+      #
+      def refresh!(view, start_time = 'NULL', end_time = 'NULL')
```

Comment:
We should pick view from view_name column, I'd only allow them to pass start and end times here